.buildkite/pipeline.yml

```
@@ -102,7 +102,16 @@ steps:
   #     ./result/bin/baseDAO-test --nettest-run-network
   #       --pattern '\$1 == "On network" || \$NF == "On network" || \$0 ~ /.On network./'
 
+  - label: check typescript api
+    depends_on:
```

Perhaps this too should run on master only after a PR is merged and not in a scheduled run?
```suggestion
    if: *not_scheduled
    depends_on:
```